core-bundle/src/Messenger/MessageHandler/BackendSearch/ReindexMessageHandler.php

```diff
@@ -23,14 +24,20 @@
 #[AsMessageHandler]
 class ReindexMessageHandler
 {
-    public function __construct(private readonly BackendSearch $backendSearch)
-    {
+    public function __construct(
+        private readonly BackendSearch $backendSearch,
+        private readonly Jobs $jobs,
+    ) {
     }
 
     public function __invoke(ReindexMessage $message): void
     {
         // Cannot run in a web request.
         if (ScopeAwareMessageInterface::SCOPE_CLI !== $message->getScope()) {
+            if ($message->getReindexConfig()->getJobId() && ($job = $this->jobs->getByUuid($message->getReindexConfig()->getJobId()))) {
```

Comment:
This is pretty cool - now you will actually see why your job got ignored if you somehow manage to trigger the job (disabled by default if no minutely cron is running but it's doable if you know how anyway 😉 ).